app.go

```diff
@@ -883,11 +882,6 @@ func (app *App) Test(req *http.Request, msTimeout ...int) (resp *http.Response,
 		return nil, err
 	}
 
-	// adding back the query from URL, since dump cleans it
-	dumps := bytes.Split(dump, []byte(" "))
```

Comment:
I think the description and title of the PR should be filled in according to [pull request template ](https://github.com/gofiber/fiber/blob/master/.github/pull_request_template.md).

Reply:
Sorry about that!!!